podcastparser.py

```diff
@@ -185,6 +185,23 @@ class EpisodeAttrFromUrl(EpisodeAttrFromHref):
     ATTRIBUTE = 'url'
 
 
+class EpisodeAttrNumber(EpisodeAttr):
+    def end(self, handler, text):
+        value = self.filter_func(text)
+        if not value.isdigit():
+            return
+        episode_num = int(value)
+        if episode_num > 0:
+            handler.set_episode_attr(self.key, episode_num)
```

Comment:
Should it be `>= 0`? I guess podcasts could have like a "announcement episode 0" kind of thing where the "real" episode is 1, but episode 0 is still a thing.

Reply:
(or is iTunes spec saying that it must be > 0? in which case this would be just fine...)